index.html

```diff
@@ -959,19 +959,24 @@ <h2><dfn>Coalesced events</dfn></h2>
                 <figcaption>Example of a curve in a drawing application – using only the coalesced
                 coordinates from <code>pointermove</code> events (the grey dots), the curve is noticeably
                 angular and jagged; the same line drawn using the more granular points provided by
-                <code>getCoalescedEvents</code> (the red circles) results in a smoother approximation
+                <code>getCoalescedEvents()</code> (the red circles) results in a smoother approximation
                 of the pointer movement.</figcaption>
             </figure>
 
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
-            zero or more <code>PointerEvent</code>s). If a trusted event is a <code>pointermove</code> or
-            <code>pointerrawupdate</code> event, the list is a sequence of all <code>PointerEvent</code>s
+            zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
+            <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
             that were coalesced into this event; otherwise it is an empty list.
             Untrusted events have their <a>coalesced events list</a> initialized to the value passed to the
             constructor.</p>
 
-            <p>The events in the coalesced events list of a trusted event will have increasing
-            {{Event/timeStamp}}s, so the first event will have the smallest {{Event/timeStamp}}.</p>
+            <p>The events in the coalesced events list of a trusted event will have:</p>
+            <ul>
+                <li>Increasing {{Event/timeStamp}}s, so the first event will have the smallest {{Event/timeStamp}}.</li>
```

Comment:
Awesome, the bullets make the description so clear!

- In particular, I can now see that the original description here was incomplete: the list will have monotonically increasing timestamps, and the parent event's timestamp will be be greater than all of them.

- Related note: perhaps we can make the first bullet in Sec10.1 and the first in Sec10.2 consistent with each other?

Reply:
yeah, big fan of bullet lists myself where appropriate. i'll try to harmonise the wording/explanation in the first bullet of the two lists.

Reply:
done